admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+= Aggregating Logs with the EFK Stack
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+Application developers can aggregate and view logs for a range of services using
+the EFK stack, which is a modified version of the
+https://www.elastic.co/videos/introduction-to-the-elk-stack[ELK stack]. This
+topic shows you how to deploy the stack using a template.
+
+Three components make up the EFK logging stack:
+
+* Elasticsearch: an object store where all logs are fed.
+* Fluentd: gathers logs from nodes and feeds them to Elasticsearch.
+* Kibana: a web UI for Elasticsearch.
+
+== Preperation for Deploying the EFK Stack
+
+Deploying the EFK stack requires:
+
+* A 'logging-deployer' secret
+* A 'logging-deployer' service account with the secret and create privileges
+* Sufficient volumes for ElasticSearch cluster storage
+* Optionally, a key and certificate for the Kibana server
+
+The deployer generates the objects necessary for communication within the
+cluster and defines secrets and templates to implement aggregated logging.
+
+. Create a new project. Once implemented in a single project, the EFK stack will apply to every project within your OpenShift cluster:
```

Comment:
"will apply to" would probably be clearer as "will collect logs for". The former makes it sound like each project has some kind of attribute set, which is not the case.